src/common/viz/CustomConfigWidgets.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "name" : "BrowserAssetWidget",
```

Comment:
As a minor point, I would probably rename this since all widgets are only used in the browser.